debian/changelog

```diff
@@ -1,18 +1,5 @@
-frostfs-node (0.34.0~97-dirty~g42554a92-jammy) jammy; urgency=medium
```

Comment:
When do we need to change this number? Every release of a `frostfs-node`? On every non-compatible change?